### ISSUE TITLE
Update spinal labeling

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/spinalcordtoolbox#3952]
A group in Calgary has improved the spinal labeling of the PAM50 template. An article is under review, but the label files are already available at https://github.com/PhillipsLab/pam50.

It would be nice to update our current spinal level files with the new ones.

A few considerations:
- We would ideally integrate this change in a stable release _before_ the article is published. Publication date is still unclear, but likely early 2023 (which would correspond to v5.9).
- Our users using these labels need to be informed that they have changed, otherwise their analysis results will change without them knowing why. 
- Based on the previous point, should we release under v6.0 (because major cross-compatibility break). Note, however, that I _don't think_ many people are using this spinal level. But I could be wrong...

**Comment by joshuacwnewton** [comment]
> An article is under review

Is there an arxiv/preprint copy of the article, by chance? I'm curious how exactly the files were improved. :eyes:

**Comment by joshuacwnewton** [comment]
Just to check... As per https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/1192, if we're updating the spinal level files, then we would also have to update the cord file too, no?

**Comment by jcohenadad** [comment]
> Just to check... As per https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/1192, if we're updating the spinal level files, then we would also have to update the cord file too, no?

I don't think so, because the spinal level files are only single voxels, and are independent from the PAM_levels file (which is a 'colored' segmentations, hence _does_ depend on the PAM50_cord). So, in this case, changing the spinal levels won't change PAM50_levels
